controllers/posting.js

```diff
@@ -21,4 +21,15 @@ async function create(req, res, next) {
         next(error)
     }
 }
-module.exports = { get, create }
+
+const getPost = async (req, res, next) => {
+    try {
+        const posting = await postingService.getPosting(req)
+        res.json({ posting: posting })
```

Comment:
as service never returns the response, it just returns data
this should just be
```
return posting
```